src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/ProviderBase/DbConnectionPoolGroup.cs

```diff
@@ -189,7 +189,7 @@ internal DbConnectionPool GetConnectionPool(DbConnectionFactory connectionFactor
             DbConnectionPool pool = null;
             if (null != _poolGroupOptions)
             {
-                Debug.Assert(ADP.IsWindowsNT, "should not be pooling on Win9x");
+                Debug.Assert(ADP.s_isWindowsNT, "should not be pooling on Win9x");
```

Comment:
Is Win9x supported by this library? If not do we need the assert? and if we don't can get get rid of IsWindowsNT since everything we support is windows nt based?